.github/workflows/pd-tests.yaml

```diff
@@ -27,19 +27,27 @@ jobs:
       matrix:
         include:
           - worker_id: 1
-            name: 'Unit Test(1)'
+            name: 'Unit Test(core)'
           - worker_id: 2
-            name: 'Unit Test(2)'
+            name: 'Unit Test(schedule)'
           - worker_id: 3
-            name: 'Tools Test'
+            name: 'Unit Test(others)'
           - worker_id: 4
-            name: 'Client Integration Test'
+            name: 'Tests(core)'
           - worker_id: 5
-            name: 'TSO Integration Test'
+            name: 'Tests(others)'
           - worker_id: 6
-            name: 'MicroService Integration Test'
+            name: 'Tools Test'
+          - worker_id: 7
+            name: 'Client Integration Test'
+          - worker_id: 8
+            name: 'TSO Integration Test'
+          - worker_id: 9
+            name: 'MicroService Integration(Core)'
```

Comment:
What does the `Core` here mean?

Reply:
Oh, I forgot to change that, changed to `Unit Test(X)`
Categorization based only on test time, which is determined by the eye ...Do you have any good ideas?